### PR TITLE
read the seconds to check for key space changes from config file, als…

### PR DESCRIPTION
**Pull request by sredxny**
## Description
Allow users to set the number of seconds before GW check again for key space changes

## Related Issue
https://github.com/TykTechnologies/tyk/issues/3058

## Motivation and Context
give solution to https://github.com/TykTechnologies/tyk/issues/3058 and accomplish one of the requisites to build the MDCB testing framework

## How This Has Been Tested
- Starts a MDCB setup
- Set 1 seconds  as value of this field
- Restart slaved GW
- Check logs, we must see every second a record saying that its checking for key space changes

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If pulling from your own
      fork, don't request your `master`!
- [x] Make sure you are making a pull request against the **`master` branch** (left side). Also, you should start
      *your branch* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] When updating library version must provide reason/explanation for this update.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] Check your code additions will not fail linting checks:
  - [x] `go fmt -s`
  - [x] `go vet`

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -167,6 +167,7 @@ type SlaveOptionsConfig struct {
 	CallTimeout                     int    `json:"call_timeout"`
 	PingTimeout                     int    `json:"ping_timeout"`
 	RPCPoolSize                     int    `json:"rpc_pool_size"`
+	SecondsToCheckKeySpaceChanges   int    `json:"seconds_to_check_key_space_changes"`
```
Comment (buger) [inline config/config.go:170]: Pls name it `keyspace_sync_interval`

Comment (sredxny) [reply config/config.go]: Done! currently there is a test that doesn't pass but is not related to this ticket. (Hostchecker test)

File: cli/linter/schema.json [hunk 2]
```diff
@@ -735,6 +735,9 @@
         },
         "rpc_pool_size": {
           "type": "integer"
+        },
+        "key_space_sync_interval": {
+          "type": "integer"
```
Comment (buger) [inline cli/linter/schema.json:740]: I wonder if you want make it float, so tests will be faster?

Comment (sredxny) [reply cli/linter/schema.json]: @buger that's a very good idea. I just pushed that change, tested and its ok

**Comment by sredxny** [comment]
Documentation https://github.com/TykTechnologies/tyk-docs/pull/1386

**Pull request merged**